server/utils/ZipHelper.ts

```diff
@@ -107,15 +110,21 @@ export default class ZipHelper {
               {
                 type: "nodebuffer",
                 streamFiles: true,
+                compression: "DEFLATE",
+                compressionOptions: {
+                  level: 5,
+                },
+                ...options,
               },
               (metadata) => {
-                const percent = Math.round(metadata.percent);
-                if (percent !== previousMetadata.percent) {
+                if (metadata.currentFile !== previousMetadata.currentFile) {
```

Comment:
Just reducing the amount of debug logging here, a bit excessive.

<img width="1094" alt="image" src="https://github.com/outline/outline/assets/380914/32d3bbc6-b122-4945-be01-a196d1f7a65c">